app/models/miq_event_definition.rb

```diff
@@ -64,7 +64,7 @@ def self.import_from_hash(event, options = {})
 
   def etype
     set = memberof.first
-    raise "unexpected error, no type found for event #{name}" if set.nil?
+    _log.error("No type found for event #{name}") if set.nil?
     set
```

Comment:
@lfu Are there any other callers to `etype` that may to handling a `nil` being returned?